index.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-var Transform = require('stream').Transform
+var Transform = require('readable-stream').Transform
```

Comment:
This should be:

```js
var Transform = require('stream').Transform || require('readable-stream').Transform
```

as it was before.  i.e. use core streams if they are sufficiently up to date.